app/config/config_dev.yml

```diff
@@ -27,7 +27,7 @@ monolog:
         server_log:
             type:   server_log
             process_psr_3_messages: false
-            host: 0:9911
+            host: localhost:9911
```

Comment:
I take that back. While it helps the dev environment not hang completely, windows will not write to 0.0.0.0 (rendering the `server:log` command useless). On Linux does that just mean write to all interfaces/IPs on the specified TCP port in this context?